packages/engine/lib/execution/src/runner/python/requirements.txt

```diff
@@ -2,4 +2,4 @@ pyarrow == 0.17
 flatbuffers == 1.12
 pynng == 0.6.2
 Cython == 0.29.21
-numpy == 1.21.1 # Pyodide is 1.15.1, but we need this
\ No newline at end of file
+numpy == 1.22.0 # Pyodide is 1.15.1, but we need this
```

Comment:
Can we check the integration tests use this? It seems like we overrode the version for a specific reason and I'm not sure our testing covers this right now

Reply:
I'm not sure. If you are unsure if we *need* 1.21.1 then we probably should just stick to our current version?